tests/test_fieldset.py

```diff
@@ -101,7 +101,7 @@ def test_fieldset_from_file_subsets(indslon, indslat, tmpdir, filename='test_sub
 
 @pytest.mark.parametrize('indstime', [range(2, 8), [4]])
 def test_moving_eddies_file_subsettime(indstime):
-    fieldsetfile = path.join(path.dirname(__file__), pardir, 'examples', 'MovingEddies_data', 'moving_eddies')
+    fieldsetfile = path.join(path.dirname(__file__), pardir, 'parcels', 'examples', 'MovingEddies_data', 'moving_eddies')
```

Comment:
This is (of course) fine. I have a comment which I wanted to make earlier but forgot at the time, though:

The moving eddies example data seem to have been generated using a Python script anyway. Would it make sense to generate the data for the test instead of downloading it?

Reply:
Actually, now with the new pathdir, the Travis CI fails, because it can't find the files anymore for `test_fieldset.py`.  So this is clearly not the solution..

Reply:
Would it make sense to log the location where examples data has been written to a new `config.yml` file, using the python `ConfigParser` class? So that different examples, tests, scripts etc can always find where the example data is downloaded and stored? Or is that overkill?